src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -877,10 +877,12 @@ def _find_using_common_tenant(self, access_token, resource):
         tenants = client.tenants.list()
         for t in tenants:
             tenant_id = t.tenant_id
-            if not hasattr(t, 'display_name'):  # Available since 2018-06-01
-                t.display_name = ""
-            if hasattr(t, 'additional_properties'):  # remove this line once SDK is fixed
-                t.display_name = t.additional_properties['displayName']
+            # display_name is available since /tenants?api-version=2018-06-01,
+            # not available in /tenants?api-version=2016-06-01
+            if not hasattr(t, 'display_name'):
+                t.display_name = None
+            if hasattr(t, 'additional_properties'):  # Remove this line once SDK is fixed
```

Comment:
Personally, I prefer `if hasattr(t, 'additional_properties') and hasattr(t.additional_properties, 'displayName'):` to make it clearer.

Reply:
`hasattr` doesn't work for `dict`, we need to use `in`. But let's release it asap. This line will be removed later anyway.